site/en/guide/data_performance.ipynb

```diff
@@ -120,13 +120,13 @@
       },
       "source": [
         "Throughout this guide, you will iterate across a dataset and measure the performance.\n",
-        "Making reproducible performance benchmarks can be difficult, different factors impacting it:\n",
+        "Making reproducible performance benchmarks can be difficult. Different factors affecting reproducibility include:\n",
         "\n",
-        "- the current CPU load,\n",
-        "- the network traffic,\n",
-        "- complex mechanisms like cache, etc.\n",
+        "- The current CPU load\n",
+        "- The network traffic\n",
+        "- Complex mechanisms, such as cache\n",
         "\n",
-        "Hence, to provide a reproducible benchmark, build an artificial example."
+        "Hence, to provide a reproducible benchmark, let's build an artificial example."
```

Comment:
Thank you! I propose a slight change in the structure, since we're describing what the user will do ("get a benchmark") and how ("you will build an example" in several steps that follow this section).

```
To get a reproducible benchmark, you will build an artificial example.
```